cmd/containerd/main.go

```diff
@@ -120,11 +120,13 @@ high performance container runtime
 		)
 		switch runtime {
 		case "runc":
+			logrus.Info("containerd: run with runc executor")
```

Comment:
this can be moved to L121 as `logrus.Infof("containerd: run with %s executor", runtime)`.

Also, it would be better if "containerd:" prefix can be moved to somewhere else (see https://github.com/sirupsen/logrus#formatters)